Doc/library/asyncio-queue.rst

```diff
@@ -1,41 +1,41 @@
 .. currentmodule:: asyncio
 
+
+======
 Queues
 ======
 
-**Source code:** :source:`Lib/asyncio/queues.py`
-
-Queues:
+asyncio queues are designed to be similar to classes of the
+:mod:`queue` module.  Although asyncio queues are not thread-safe
```

Comment:
Although asyncio queues are not thread-safe, they are designed to be used specifically in async/await code.